.github/workflows/promote.yaml

```diff
@@ -44,4 +44,4 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           destination-channel: ${{ inputs.track-name }}/${{ env.promote-to }}
           origin-channel: ${{ inputs.track-name }}/${{ env.promote-from }}
-          charmcraft-channel: ${{ inputs.track-name }}/stable
+          charmcraft-channel: latest/stable
```

Comment:
Is this intentional? We probably don't want to use `latest` at all

Reply:
Yes, that was intentional. That's the channel for the `charmcraft` snap. We also use latest in other workflows for it. 
Pinning the version here would be problematic, because none of our automatic helpers can update that.